draft-ietf-suit-architecture.md

```diff
@@ -671,15 +674,15 @@ implement the following functionality:
    digital signature. (Alternatively, access to a key store for use
    with the message authentication code.)
 
--  Ability to expose boot process-related data to the application
+-  There must be an ability to expose boot process-related data to the application
    firmware (such as to the status tracker).  This allows
-   to share information about the current firmware version, and the
+   sharing information about the current firmware version, and the
    status of the firmware update process and whether errors have occurred.
 
 -  Produce boot measurements as part of an attestation solution. See
    {{I-D.ietf-rats-architecture}} for more information. (optional)
 
--  Ability to decrypt firmware images, in case confidentiality protection
+-  The bootloader must be able to decrypt firmware images, in case confidentiality protection
```

Comment:
These were an attempt to keep the list structure paralle, but I'm not sure I quite got it right for all the list items.

Reply:
I think this works.